Makefile

```
@@ -91,15 +91,16 @@ run:
 	@concurrently "make run-db" "make run-app"
 
 init-db:
-	TREECREEPER_SCHEMA_DIRECTORY=example-schema packages/tc-api-db-manager/index.js
+	NEO4J_BOLT_URL=${LOCAL_BOLT_URL} TREECREEPER_SCHEMA_DIRECTORY=example-schema packages/tc-api-db-manager/index.js
 
 cypress-page:
 	start-server-and-test "make run-app" http-get://localhost:8888/MainType/create "make cypress-run-page"
 
 cypress-primitives:
 	start-server-and-test "make run-app" http-get://localhost:8888/MainType/create "make cypress-run-primitives"
 
-
+run-biz-ops: build-statics
+	NEO4J_BOLT_URL=${BIZ_OPS_BOLT_URL} TREECREEPER_SCHEMA_URL=${SCHEMA_BASE_URL} SCHEMA_UPDATE_Mode=poll PROJECT_NAME=biz-ops-api PRODUCT_NAME=biz-ops nodemon --inspect demo/api.js
```

```suggestion
	NEO4J_BOLT_URL=${BIZ_OPS_BOLT_URL} TREECREEPER_SCHEMA_URL=${SCHEMA_BASE_URL} PROJECT_NAME=biz-ops-api PRODUCT_NAME=biz-ops nodemon --inspect demo/api.js
```

I guess you don't ned to pass in update mode as it defaults in the library now?